rebuild/test/unit/Signature.test.ts

```diff
@@ -0,0 +1,80 @@
+import {expect} from "chai";
+import {BLST_CONSTANTS, CoordType, SecretKey, Signature} from "../../lib";
+import {expectEqualHex, expectNotEqualHex, sullyUint8Array} from "../utils";
+import {KEY_MATERIAL, invalidInputs, validSignature} from "../__fixtures__";
+
+describe("Signature", () => {
+  it("should exist", () => {
+    expect(Signature).to.exist;
+  });
+  describe("constructor", () => {
+    it("should have a private new Signature()", () => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-return
+      expect(() => new (Signature as any)()).to.throw("Signature constructor is private");
+    });
+    describe("Signature.deserialize()", () => {
+      it("should take uncompressed byte arrays", () => {
+        expectEqualHex(
+          Signature.deserialize(validSignature.uncompressed).serialize(false),
+          validSignature.uncompressed
+        );
+      });
+      it("should take compressed byte arrays", () => {
+        expectEqualHex(Signature.deserialize(validSignature.compressed), validSignature.compressed);
+      });
+      it("should create jacobian or affine points", () => {
+        expectEqualHex(Signature.deserialize(validSignature.compressed, CoordType.affine), validSignature.compressed);
```

Comment:
Signature.deserialize() should give Signature class, which hasn't been serialized, can't understand how equal hex comparison can be done

Reply:
`expectEqualHex` will serialize a `Serializable` object.  Its a bit confusing though. I will update to show the serialize getting called and remove that condition from `expectEqualHex`

Reply:
hmmm so `toHexString` in it serializes it to hex, not sure its intuitive enough, were previous tests written like this?